frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/object-service-dashboards.spec.ts

```diff
@@ -0,0 +1,134 @@
+const testName = `test-${Math.random()
+  .toString(36)
+  .replace(/[^a-z]+/g, '')
+  .substr(0, 5)}`;
+
+const testBucket = {
+  apiVersion: 'objectbucket.io/v1alpha1',
+  kind: 'ObjectBucketClaim',
+  metadata: {
+    namespace: 'openshift-storage',
+    name: testName,
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+const installStorageCluster = () => {
+  cy.login();
+  cy.visit('/');
+  cy.install();
+};
+
+describe('Tests Buckets Card', () => {
```

Comment:
You don't need multiple describe blocks. Just use one describe block and separate different cards into various 'it' blocks.

Reply:
Do you mean that I should covert existing additional describe blocks into it blocks and remove the sub-it blocks that are inside those? I've pushed a patch that simply removes the additional describes but keeps their it blocks.

Let me know what changes you have in mind and I'll push them.